assigner/baserepo.py

```diff
@@ -436,18 +470,3 @@ def name(cls, semester, section, assignment, user):
     def push(self, base_repo, branch="master"):
         """Push base_repo code to this repo"""
         base_repo.push_to(self, branch)
-
-
-if __name__ == "__main__":
```

Comment:
I assumed this wasn't being used and was for testing / from the old one file assigner.
@LinuxMercedes @michaelwisely  confirm?

Reply:
Yep, that all can go!